bin/common/src/metabuild_common/core.clj

```diff
@@ -5,7 +5,7 @@
             [metabuild-common.files :as files]
             [metabuild-common.input :as input]
             [metabuild-common.misc :as misc]
-            [metabuild-common.output :as output]
+            [metabuild-common.output :as out]
```

Comment:
i'm surprised to see this change. The alias `output` seemed a better consistent choice.

Reply:
this seems the opposite of the `sh` -> `shell` below.

Reply:
no bikeshedding. i'm not looking for all of these to match my expectations. cohesive whole is the goal, not my personal. I was just surprised.